admin_guide/sysctls.adoc

```diff
@@ -113,11 +113,16 @@ launch.
 == Enabling unsafe sysctls
 
 The cluster administrator can allow certain unsafe sysctls for very special
-situations such as high-performance or real-time application tuning.
+situations such as high performance or real-time application tuning.
 
 If you want to use unsafe sysctls, cluster administrators must enable them
 individually on nodes. They can enable only namespaced sysctls.
 
+You can further control which sysctls can be set in pods by specifying lists of sysctls or sysctl patterns in the `forbiddenSysctls` and/or `allowedUnsafeSysctls` fields of the Security Context Constraints.
```

Comment:
Per ISG, drop `/or` here:
and/or -> and

"Do not use the construction and/or, which can cause confusion and problems for translation.
Instead, clarify the options by using and or or, or by stating the choices, such as
option A, option B, or both."

Reply:
Hey @bobfuru. I appreciate the guidance. I updated the text.